aip/general/0193.md

```diff
@@ -62,11 +62,90 @@ a [`PreconditionFailure`][PreconditionFailure].
 **Note:** `ErrorInfo` represents a special case. There **must** be exactly one
 `ErrorInfo`. It is required.
 
-### Localization
+### Error messages
 
-Error messages **must** be in English. If a localized error message is also
-required, the service **should** use [`google.rpc.LocalizedMessage`][details]
-in the `details` field.
+The [`google.rpc.Status`][Status] error payload **should** contain two
+error messages.
+
+1. The payload **must** contain an error message in the `message` field
+   found at the root of [`Status`][Status].
+1. The payload **should** contain
+   [`google.rpc.LocalizedMessage`][LocalizedMessage] in `details` on
+   `Status`. This is the location of the second error message.
+
+The error message on `Status` should,
+
+- Be developer-facing, human-readable, and in English
+- Both expalin the error and offer an actionable resolution to it
+  ([citation](https://cloud.google.com/apis/design/errors#error_model))
+
+Once launched the value of `message` at the root-level of `Status`
+**must not** change for the error payload to be backwards compatible.
+
+When including `LocalizedMessage`, both the `locale` and `message`
+fields **must** be present.
+
+#### Changing error messages
+
+A new, additional error message **may** be added because the error
+message at the root of the payload **must not** change once launched.
```

Comment:
```suggestion
message at the root of the payload cannot be changed once launched.
```
I think this guidance is already provided and here it seems to be the motivating reason, not guidance.

I don't feel strongly about this comment though.